src/pontibus/tests/utils/test_settings.py

```diff
@@ -38,7 +38,7 @@ def test_positives(self, settings, attr, value):
 
 
 class TestPackmolSolvationSettings:
-    @pytest.fixture(scope="class")
```

Comment:
this was causing issues with mutated state between tests w/ pydantic v2. it's pretty lightweight so removing this scope isn't a big deal imo.